diff.go

```diff
@@ -311,15 +313,17 @@ func diffComponentEnv(oldDef, newDef ComponentDefinition, componentName Componen
 	return diffInfos
 }
 
-func diffComponentVolumes(oldDef, newDef ComponentDefinition, componentName ComponentName) []DiffInfo {
-	diffInfos := []DiffInfo{}
+func diffComponentVolumes(oldDef, newDef ComponentDefinition, componentName ComponentName) DiffInfos {
+	diffInfos := DiffInfos{}
```

Comment:
Yesterday Marian asked about how we deal with a change in volume size.
This is not picked up now as a separate diff type. Consider if that is possible and wise.